src/primitives/line.jl

```diff
@@ -19,3 +19,14 @@ Line(a::Tuple, b::Tuple) = Line(Point(a), Point(b))
 paramdim(::Type{<:Line}) = 1
 
 (l::Line)(t) = l.a + t * (l.b - l.a)
+
+function Base.in(p::Point, l::Line)
+  w = norm(l.b-l.a)
+  d = evaluate(Euclidean(), p, l)
+  # d ≈ 0.0 will be too precise, and d < atol{T} can't scale.
+  d+w ≈ w
+end
+
+function Base.:(==)(l1::Line, l2::Line)
+  l1.a in l2 && l1.b in l2 && l2.a in l1 && l2.b in l1
+end
```

Comment:
```suggestion
==(l1::Line, l2::Line) = l1.a ∈ l2 && l1.b ∈ l2 && l2.a ∈ l1 && l2.b ∈ l1
```

Reply:
I copied pasted the `\in` operator in Julia on GitHub and it may have changed the character to some unicode that is invalid Julia. Appreciate if you can double check after accepting  the suggestion.